packages/shorebird_cli/lib/src/commands/patch/ios_patcher.dart

```diff
@@ -210,12 +213,17 @@ class IosPatcher extends Patcher {
     }
 
     final patchFileSize = patchFile.statSync().size;
+    final privateKeyFile = argResults.file(CommonArguments.privateKeyArgName);
+    final hash = sha256.convert(patchBuildFile.readAsBytesSync()).toString();
+    final hashSignature = (hash, privateKeyFile).signature;
```

Comment:
```suggestion
    final hashSignature = sign(hash, privateKeyFile);
```
minor but it would read a bit better if there was a `sign` method instead of `signature` extension on on a record type.